packages/react-scripts/package.json

```diff
@@ -21,6 +21,9 @@
     "utils",
     "backpack-addons"
   ],
+  "scripts": {
+    "test:addons": "jest --testPathPattern=backpack-addons"
```

Comment:
This will also run `babelIncludePrefixes.test.js`, the only other test in the folder other than the kitchen sink fixtures. Very open to any alternatives here.